### ISSUE TITLE
Difference with original autodE questionable 

### ISSUE THREAD

**Issue by eneas77** [chimie-paristech-CTM/TS-tools#5]
Hi,

After reading your preprint I can´t find a real difference of TS-tools from autodE, considering that the core app is autodE (xtb+Gaussian). AutodE is capable to accept smiles representations (both individual molecules and reactions) as inputs, so I´m completely lost. Specifically:

1) What is the innovation gain from existing autodE software
2) What is the simplification from autodE input set-up methodology
3) autodE can manage solvent environments invoking them as it is stated in the documentation (as well as temperature) so what is the add-on compared with it.
4) What is the utility of reinventing the wheel to study tri-molecular reactions when the program core is basically autodE+dependencies. You can test the same just with autodE.

In summary, a bit sceptical about TS-tools utility and publishing a preprint of something already done.

E77

**Comment by tstuyver** [comment]
Dear Eneas,

As you may have read in the pre-print, there is a whole cottage industry of automated TS search approaches, autodE is one of them -- and a really good one -- but far from the only one out there. As much as you may think its methodology is superficially the same as the methodology of TS-tools, it simply is not (we use reactant alignment as well as reactive AFIR paths -- choices that were inspired by other TS search codes, but not by autodE, which uses NEB among others). One specific issue for which our approach is ideally suited is indeed tri-molecular reactions. You cannot currently run them with autodE, and there is no obvious way to make this possible (trust me, we tried, and we also discussed this with the original developers of autodE). Other issues we struggled with in autodE is for example maintaining a consistent stereochemistry; in reactive paths that seems to be preserved by default. All that being said, I want to underscore here that I love autodE, and have used it extensively in my own work so far. I absolutely agree that for most applications, there is no obvious reason to pick TS-tools and you may want to stick to autodE. However, I do not consider it wrong to open up new tools in a user-friendly packaging to the community. I put in the work to make this tool in-house so that we can easily study tri-molecular reactions as part of another project, and I consider it a service to the community to make it publicly available when it turns out to work neatly. It is up to you to decide whether you want to try it or not.
